vignettes/ChromBand.Rmd

```diff
@@ -0,0 +1,379 @@
+
+```{r basicParams}
+params <- new("ChrMapHyperGParams",
+              conditional=FALSE,
+              testDirection="over",
+              universeGeneIds=entrezUniverse,
+              geneIds=selectedEntrezIds,
+              annotation="hgu95av2",
+              pvalueCutoff=0.05)
+paramsCond <- params
+paramsCond@conditional <- TRUE
+```
+
+The test computations are performed by
+
+```{r basicTest}
+hgans <- hyperGTest(params)
+hgansCond <- hyperGTest(paramsCond)
+```
+
+The results can be summarized by
+
+```{r result1}
+sumUn <- summary(hgans, categorySize=1)
+chrSortOrder(sumUn)
+
+sumCond <- summary(hgansCond, categorySize=1)
+chrSortOrder(sumCond)
+```
+
+For the standard test, the structure of the chromosome band graph is ignored and a Hypergeometric test is applied to each band independently. For the conditional test, the hierarchical relationship among the bands as represented by the graph is used in the computation. The highest-resolution bands (those with no children in the graph) are tested first. Testing proceeds with the bands whose children (sub-bands) have already been tested. For these bands, the gene annotations that are inherited from significant child nodes (children with $p$-value smaller than the specified cutoff) are removed prior to testing to yield a conditional test.
+
+The effect of the conditional test is illustrated by examining the results for 14q and its sub-bands. In the standard test, we see that 14q22 and 14q22.2 both have a significant $p$-value. In the conditional test, only 14q22.2 remains. The conclusion is that there is not enough additional evidence beyond that provided by 14q22.2 to mark 14q22 as significant.
+
+# GSEA using linear models
+
+GSEA is a popular method that can be used to assess whether or not particular gene sets are associated with a phenotype of interest [@Subramanian2005; @Tian2005; @JiangGent2007]. Most applications of this method do not explicitly deal with structure of the gene sets, but when analyzing chromosomal location such methods are desirable. We present a simple approach that is similar in spirit to traditional GSEA, and generalizes nicely to accommodate nested categories. Consider the situation where gene $i$ has an associated measure of differential expression $y_i$; for example, an attenuated $t$-statistic derived from a differential expression analysis such as [limma]{.sans-serif} [@limma]. Given a particular category, GSEA asks whether the distribution of $y_i$-s restricted to the category of interest is "unusual". Thus, in<span><a href="#fig:gseaStripplot"> Figure 2</a><span>, we might be interested in knowing whether the distribution of $y_i$ values for genes in 12q21 is different from that of the genes in 12q (for a local test) or of all genes in the gene universe (for a global test).<a href="#fig:gseaStripplot"> Figure 2</a> is produced by
```

Comment:
- [ ] Make the link for `limma`
- [ ] Remove the `<span></span>` tags.

Reply:
but you said you want the anchor to link, it is the html syntax for an inline anchor
, please see the html syntax.

Reply:
\@ref  was what I was up all night trying to fid solution , it never worked , I looked everywhere for the solution including this documentations never worked,I only used the one that worked. Can you just show me the code syntax